src/Apps/Artwork/Components/ArtworkImageBrowser/UtilButton.tsx

```diff
@@ -126,7 +126,7 @@ const UtilButtonInnerText: React.FC<UtilButtonInnerTextProps> = ({
     return null
   }
 
-  if (longestLabel) {
+  if (longestLabel && process.env.NODE_ENV !== "test") {
```

Comment:
we hide the component that helps us calculate the maximum possible size of the button with a long text, since this functionality is not needed for tests and will add some problems (e.g. `screen.getByText("Label")` will return an error, since there will be 2 elements with the text "Label" on the page)